hassrelease/changelog.py

```diff
@@ -92,7 +96,7 @@ def generate(release, prs):
         pr = prs.get(line.pr)
 
         if (pr.milestone is not None and
-                pr.milestone.title != release.version_raw):
+                pr.milestone.title != release.version):
```

Comment:
This makes no sense. `milestone.title` is a string, `release.version_raw` is a string. `release.version` is not a string.